tools/integration_tests/util/file_operations/file_operations.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
```

Comment:
Can we rename this package from file_operations to operations ? We can have file_operations.go and later dir_operations.go in operations package.